apps/api/src/app/environments-v1/usecases/output-renderer-usecases.ts

```diff
@@ -0,0 +1,143 @@
+// Base interface for all renderers
+import { render } from '@maily-to/render';
+import {
+  ChatRenderOutput,
+  EmailRenderOutput,
+  InAppRenderOutput,
+  PushRenderOutput,
+  RedirectTargetEnum,
+  SmsRenderOutput,
+  TipTapNode,
+} from '@novu/shared';
+import { z } from 'zod';
+import { ExpendEmailEditorSchemaUseCase } from '../../environments/render/email-schema-extender';
+
+class RenderCommand {
+  controlValues: Record<string, unknown>;
+}
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** The existing convention is to have the command in it's own file, it should also extend from a sub-class of `BaseCommand` to ensure that validation is enforced in the use-case layer. Please:
* Move the command to it's own file
* Extend from `BaseCommand`